### PR TITLE
Merge into UCLALibrary master

### PR DESCRIPTION
**Pull request by DRickard**

### CODE REVIEW

**Comment by akohler** [comment]
@DRickard Lots of changes!  I'm not sure all are correct, though.  I noticed Application.cfm has references added for SIA (APPLICATION.dsn), for example, which I suspect is incorrect.

So, please take a look through the changed files
https://github.com/UCLALibrary/public_service_statistics/pull/1/files

If things don't look right, you should be able to fix them on your local, push up to your fork, and update this PR - or we can close this PR and you can issue a new one.

File: reference/refForm.cfm [hunk 1]
```diff
@@ -238,8 +238,8 @@ Did you provide a referral to the patron?&nbsp;&nbsp;
 </cfoutput>
 </div>
 <div style="background-color: #DEDEDE;">
-Scheduled Consultation? Use <a href="http://sia.library.ucla.edu" target="_blank">SIA</a>.<br/>
-<a href="http://stats.library.ucla.edu/reference/category_definitions.cfm" target="_blank">Question Type Category Definitions and Guidelines for use</a>.<br/>
+Scheduled Consultation? Use <a href="http://h-unitproj.library.ucla.edu/sia" target="_blank">SIA</a>.<br/>
```
Comment (akohler) [inline reference/refForm.cfm:241]: @DRickard This URL references h-unitproj, probably should just be unitproj.

**Comment by DRickard** [comment]
The SIA ref is legitimate: we're borrowing academic department data from SIA, and rather than keep data consistent between schemas I do one lookup from SIA.  I'll review the rest of the change log tomorrow.

David Rickard
UCLA Library Digital Initiatives and Information Technology
drickard1967@library.ucla.edu

**Comment by akohler** [comment]
@DRickard Are you ready to merge this pull request into master?

**Comment by akohler** [comment]
@DRickard : I want to close this PR.  Please confirm the changes are appropriate and I'll do the merge.

**Comment by DRickard** [comment]
Yes, the changes are confirmed; merge the update.

David Rickard
UCLA Library Digital Initiative and Information Technology
drickard1967@library.ucla.edu

**Pull request merged**